src/internal/matcher/nearly_eq.rs

```diff
@@ -0,0 +1,132 @@
+use crate::internal::fmt::MaybeDebug;
+use crate::internal::{ArgumentMatcher, MaybeDebugWrapper};
+use nameof::name_of_type;
+use nearly_eq::NearlyEq;
+use std::fmt::{self, Debug};
+
+/// Creates a new `ArgumentMatcher` that matches against values using [`NearlyEq`]
+pub fn nearly_eq<T, U>(value: T) -> NearlyEqArgumentMatcher<T, U>
+where
+    T: NearlyEq<T, U> + MaybeDebug,
+    U: MaybeDebug,
+{
+    NearlyEqArgumentMatcher {
+        value,
+        accuracy: T::eps(),
+    }
+}
+
+/// Creates a new `ArgumentMatcher` that matches against values using [`NearlyEq`]
+pub fn nearly_eq_with_accuracy<T, U>(value: T, accuracy: U) -> NearlyEqArgumentMatcher<T, U>
+where
+    T: NearlyEq<T, U> + MaybeDebug,
+    U: MaybeDebug,
+{
+    NearlyEqArgumentMatcher { value, accuracy }
+}
+
+pub struct NearlyEqArgumentMatcher<T, U>
+where
+    T: NearlyEq<T, U> + MaybeDebug,
+    U: MaybeDebug,
+{
+    value: T,
+    accuracy: U,
+}
+
+impl<T, U> Debug for NearlyEqArgumentMatcher<T, U>
+where
+    T: NearlyEq<T, U> + MaybeDebug,
+    U: MaybeDebug,
+{
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        f.debug_struct(name_of_type!(Self))
+            .field("value", &MaybeDebugWrapper(&self.value))
```

Comment:
Use `nameof` for field names

Reply:
Addressed in https://github.com/myelin-ai/mockiato/pull/99/commits/a54d92b9744d572ff2a9363001fba2955bef58e1